cmd/perfguard/runner.go

```diff
@@ -288,16 +288,8 @@ func (r *runner) Run() error {
 
 	timeElapsed := time.Since(startTime)
 
-	if !r.args.quiet {
-		if r.heatmap != nil && r.stats.affectedSampleTime != 0 {
-			fmt.Fprintf(r.stderr, "Affected samples time: %s\n", r.stats.affectedSampleTime)
-		}
-		suffix := "auto-fixable"
-		if r.autofix {
-			suffix = "fixed"
-		}
-		fmt.Fprintf(r.stderr, "Found %d issues (%d %s)\n",
-			r.stats.issuesTotal, r.stats.issuesFixable, suffix)
```

Comment:
Are you sure that moving this code from the runner methods won't make optimize mode become silent?
Run is used from both lint and optimize modes if I remember it correctly.

Reply:
+